templater/runner/src/test/scala/coop/rchain/perf/DeployProposeSimulation.scala

```diff
@@ -42,11 +46,13 @@ class DeployProposeSimulation extends Simulation {
   val protocol = RNodeProtocol(host, port)
 
   val scn = scenario("DeployProposeSimulation")
-    .repeat(2) {
-      exec(deploy(contract))
-        .pause(1)
-        .exec(propose())
-        .pause(1)
+    .foreach(List(contract), "contract") {
```

Comment:
Why did you do this unrelated change?

Reply:
sadly it's not unrelated. the way a param can be passed to a scenario is via EL expressions. it's stupid, I don't like it but that's how the whole gatling framework is organised.
this change is to align with the way params are being passed.